### PR TITLE
fix: fix Arazzo json pointers and other minor issues

### LINKED ISSUE #37: Museum API Arazzo Example Issues
Hi, 

I was just going over the example Arazzo documents you have here:

- https://github.com/Redocly/museum-openapi-example/blob/main/arazzo/museum-api.arazzo.yaml
- https://github.com/Redocly/museum-openapi-example/blob/main/arazzo/museum-tickets.arazzo.yaml

And think there are a few issues in them:

I found similar issues in both documents:

```
[73:27] body reference must be empty or start with #: $response.body.eventId
[74:17] body reference must be empty or start with #: $response.body.name
[109:27] body reference must be empty or start with #: $response.body.eventId
```

From my reading of the Arazzo specification I believe body runtime expressions like this should be `$response.body#/eventId` ie the runtime expression to the body and then a json pointer to the field within.

This is based on the ABNF syntax here: https://spec.openapis.org/arazzo/latest.html#runtime-expressions

which defined the `expression` as `"$response." source`, `source` as `source = ( header-reference / query-reference / path-reference / body-reference )` and `body-reference` as `body-reference = "body" ["#" json-pointer ]`

### PR DESCRIPTION
**Pull request by adamaltman**
## What/Why/How?

This fixes a variety of issues:
- Fix Arazzo syntax to use JSON Pointer for body according to ABNF (#37 ).
- Add `x-serverUrl` examples in Arazzo.
- Update contributing file to correct for this specific repo where there are no release pipelines or docs folder.
- Streamline the PR template because one section doesn't make very much sense in this repo.
- Improve the sentence case pattern to allow for acronyms and allowing to remove the lint ignore file.
- Fix the `index.md` docs for consistency and to use the `preview` command.
- Clarify `ticketDate` description in the OpenAPI description.
- Remove `package-lock.json` as it shouldn't typically be distributed in a template type of repo.

## Reference

- Fixes #37 

## Testing

- tested in my local environment.

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: arazzo/museum-api.arazzo.yaml [hunk 1]
```diff
@@ -4,12 +4,13 @@ info:
...
     Use the Museum API with Arazzo as an example of describing multi-step workflows.
     Built with love by Redocly.
-  version: 1.0.0
+  version: 1.0.1
 
 sourceDescriptions:
   - name: museum-api
     type: openapi
     url: ../openapi.yaml
+    x-serverUrl: "http://localhost:4000/_mock/openapi" # For local development testing
```
Comment (DmitryAnansky) [inline arazzo/museum-api.arazzo.yaml:13]: Last time I added changes to this file I  intentionally updated the `openapi.yaml` with mock-server
```
servers:
  - url: "https://redocly.com/_mock/docs/openapi/museum-api"
```
  
  The main idea was that users don't need to set any mock-server on their own and Spot run will work out of the box.

**Review by DmitryAnansky — APPROVED** [review]
Looks good to me,
I only have some doubts about `x-serverUrl` example.

**Comment by adamaltman** [comment]
I want to show how `x-serverUrl` is used in practice.

File: openapi.yaml [hunk 2]
```diff
@@ -454,7 +454,7 @@ components:
       summary: Create special event
       value:
         name: Mermaid Treasure Identification and Analysis
-        location: Under the seaaa 🦀 🎶 🌊.
+        location: Under the seaaa ��� 🎶 🌊.
```
Comment (adamaltman) [inline openapi.yaml]:
```suggestion
        location: Under the seaaa 🦀 🎶 🌊.
```

**Review by RomanHotsiy — APPROVED** [review]

**Pull request merged**